src/xcode/ENA/ENA/Source/Services/Risk/KeyPackageDownload/KeyPackageDownload.swift

```diff
@@ -290,6 +311,7 @@ class KeyPackageDownload: KeyPackageDownloadProtocol {
 		switch downloadMode {
 		case .daily:
 			client.availableDays(forCountry: country) { result in
+				print("availableServerData: \(result)")
```

Comment:
Please use proper logging or remove `print`

Reply:
removed

Reply:
Merci. Should we add a swiftlint rule for this? (cc @marcussc) 
I might have one laying around from an old project…

Reply:
Nice idea 👍
But please in an extra PR :)